components/blocks/FilesToCards.vue

```diff
@@ -243,9 +271,9 @@ export default {
     sortedArray() {
       let filtered = this.filteredData;
       if (this.searchGroup.length > 0) {
-        filtered = filtered.filter((card) => {
+        filtered = filtered.filter(() => {
           if (this.searchGroup) {
-            return this.searchGroup.includes(card.group);
+            return this.searchGroup.some(r => this.cardTags.includes(r))
```

Comment:
won't this always be true? `this.cardTags` is the unique list of all card tags across all cards and searchGroup is always a subset of that.  Maybe something like `this.searchGroup.some(tag => ['tags', 'groups', 'blah'].some((tagType) => card[tagType].includes(tag)))`

Reply:
when the page firsts loads, searchGroup is empty because tags havent been selected from the dropdown menu.

Reply:
i think i need to leave if(this.searchGroup), but i changed the `some` line and filter works! thank you.